client/cypress/integration/sessions.spec.js

```diff
@@ -114,11 +95,6 @@ describe("sessions screen", () => {
     cy.visit("/sessions");
     const option = cy.get("#show-graded-checkbox");
     option.should("not.have.attr", "checked");
+    cy.get("#toggle").click();
```

Comment:
wondering if we can improve these tests by adding spies to check that graphql is called as expected? 

Rough example here: 

```js
cy.get('@post').should((req) => {
  expect(req.method).to.equal('POST')
  expect(req.url).to.match(/\/posts$/)
  // it is good practice to add message to the assertion
  expect(req, 'has duration in ms').to.have.property('duration').and.be.a('number')
})
```

...if crazy extra burden, skip. But overall be good to have spying on graphql calls a tool in our toolbox for cypress tests